pallets/band/src/lib.rs

```diff
@@ -92,7 +94,21 @@ pub use pallet::*;
 
 impl<T: Config<I>, I: 'static> DataFeed<T::Symbol, Rate, u64> for Pallet<T, I> {
     fn quote(symbol: &T::Symbol) -> Result<Option<Rate>, DispatchError> {
-        Ok(Self::rates(symbol).map(|rate| rate.into()))
+        let rate = if let Some(rate) = Self::rates(symbol) {
+            rate
+        } else {
+            return Ok(None);
+        };
+        let current_time = T::UnixTime::now().as_millis().saturated_into::<u64>();
+        let stale_period = T::GetBandRateStalePeriod::get();
+        let current_period = current_time
+            .checked_sub(rate.last_updated)
+            .ok_or(Error::<T, I>::RateHasInvalidTimestamp)?;
+        sp_std::if_std! {
+            println!("current_time: {}; current_period: {}", current_time, current_period);
```

Comment:
Looks like forgotten debug message. You can either remove it or transform into useful log message